common/src/main/java/com/wynntils/features/inventory/MythicFoundSoundFeature.java

```diff
@@ -2,15 +2,18 @@
  * Copyright © Wynntils 2023.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.features.embellishments;
+package com.wynntils.features.inventory;
 
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.ConfigCategory;
 import com.wynntils.core.features.Feature;
 import com.wynntils.models.containers.event.MythicFoundEvent;
 import com.wynntils.utils.mc.McUtils;
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.sounds.SoundEvent;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
+@ConfigCategory(Category.INVENTORY)
```

Comment:
Say what? Was this not classified? :-o Anyway, I think it belongs with the other two mythics features in a combat/gameplay category. (More general: I don't think "Loot chest stuff" belongs to "Inventory")